protocol/x/clob/keeper/orders.go

```diff
@@ -104,11 +104,12 @@ func (k Keeper) BatchCancelShortTermOrder(
 
 			if err != nil {
 				failure = append(failure, clientId)
-				log.InfoLog(
+				log.DebugLog(
 					ctx,
 					"Batch Cancel: Failed to cancel a short term order.",
 					log.Error, err,
 				)
+				telemetry.IncrCounter(1, types.ModuleName, metrics.BatchCancelFailed, metrics.Count)
```

Comment:
Nit: is metric name `BatchCancelFailed` accurate if this is incremented when *one* of the cancels on the `BatchCancel` fails?